source/conf.py

```diff
@@ -13,6 +13,7 @@
 # import os
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
+import os
```

Comment:
why is this imported

Reply:
leftover from when the env variable was checked to run headless, will be needed if we need to modify drawio_headless ourselves